rlImGui/rlImGui.cs

```diff
@@ -70,6 +71,28 @@ public static void Setup(bool darkTheme = true, bool enableDocking = false)
             EndInitImGui();
         }
 
+        /// <summary>
+        /// Set the filename of the INI file that ImGui uses to store the window configuration.
+        /// It is set to "imgui.ini" by default.
+        ///
+        /// Call this after rlImGui.Setup.
+        /// </summary>
+        /// <param name="iniFilename">The file path to store the ini file.</param>
+        public static void SetIniFilename(string iniFilename)
+        {
+            if (iniFilenameAlloc != 0)
+                Marshal.FreeHGlobal(iniFilenameAlloc);
+
+            byte[] nameBytes = System.Text.Encoding.ASCII.GetBytes(iniFilename + "\0");
+            iniFilenameAlloc = Marshal.AllocHGlobal(nameBytes.Length);
+            Marshal.Copy(nameBytes, 0, iniFilenameAlloc, nameBytes.Length);
+            
+            unsafe
+            {
+                ImGui.GetIO().NativePtr->IniFilename = (byte*) iniFilenameAlloc;
+            }
+        }
+
```

Comment:
Not sure I like this being internal to rlImGui. it's not raylib specific and doesn't really have anything to do with this backend. This code would be used on any C# imgui system.
I'd prefer to keep rlImGui just for things that deal with raylib and ImGui, not random imgui utilities, they should go into a separate and optional library.